packages/react-app/src/components/giveaways/details/MainContent.tsx

```diff
@@ -40,6 +40,7 @@ const GiveawayMainContent = ({ participants }: GiveawayMainContentProps) => {
   return (
     <Stack sx={{ paddingLeft: { sm: '80px', lg: '0px' } }}>
       <Typography
+        className="giveaway-title"
         sx={{
           fontSize: '35px',
           fontWeight: 'bold',
```

Comment:
I believe this one was missed

Reply:
Most of the sx props were intentionally not changed due to sx and style props working differently and the replacement not being as clean as just passing it className. In this case one possible solution would be to find out what css selectors we need and adjust styling accordingly, however that may generate a bigger code footprint unnecessarily. I'll double check this case either way 👍